src/orquestra/sdk/_base/_workflow.py

```diff
@@ -521,6 +582,7 @@ def workflow(
 def workflow(
     fn: Optional[Callable[_P, _R]] = None,
     *,
+    resources: Optional[_dsl.Resources] = None,
```

Comment:
**suggestion**: please describe what's the consequence of using the default value vs passing something here.